classes/battle/BattleManager.php

```diff
@@ -1053,22 +1060,26 @@ public function jutsuCollision(
     /**
      * @throws RuntimeException
      */
-    private function getEvasionStatAmount(Fighter $fighter, Jutsu $fighter_jutsu): float|int {
+    private function getEvasionStatAmount(Fighter $fighter, Jutsu $fighter_jutsu, int $target_stat_total): float|int {
```

Comment:
This function is really performing a different purpose now, we should rename
```suggestion
    private function getEvasionPercent(Fighter $fighter, Jutsu $fighter_jutsu, int $target_stat_total): float|int {
```

Reply:
Done.